gbpservice/neutron/nsf/config_agent/loadbalancer.py

```diff
@@ -0,0 +1,209 @@
+        context = kwargs.get('context')
+        del kwargs['context']
+        self.cctxt.cast(context, 'update_pool_stats',
+                        pool_id=kwargs['pool_id'],
+                        stats=kwargs['stats'],
+                        host=kwargs['host'])
+
+    def pool_destroyed(self, pool_id):
+        context = kwargs.get('context')
+        del kwargs['context']
+        self.cctxt.cast(self.context, 'pool_destroyed',
+                        pool_id=kwargs['pool_id'])
+
+    def pool_deployed(self, **kwargs):
+        context = kwargs.get('context')
+        del kwargs['context']
+        self.cctxt.cast(self.context, 'pool_deployed',
+                        pool_id=kwargs['pool_id'])
+
+
+class LbAgent(loadbalancer_db.LoadBalancerPluginDb):
+    RPC_API_VERSION = '1.0'
+    _target = target.Target(version=RPC_API_VERSION)
+
+    def __init__(self, conf, sc):
+        self._conf = conf
+        self._sc = sc
+        super(LbAgent, self).__init__()
+
+    def _prepare_request_data(self, resource, kwargs):
+
+        request_data = {'info': {
+            'version': Version,
+            'service_type': 'loadbalancer'
+        },
+
+            'config': [{
+                'resource': resource,
+                'kwargs': kwargs
+            }]
+        }
+
+        return {'request_data': request_data}
+
+    def _post(self, context, tenant_id, name, **kwargs):
+        db = self._context(context, tenant_id)
+        context.__setattr__('service_info', db)
+        kwargs.update({'context': context})
+        resource = name
+        body = self._prepare_request_data(resource, kwargs)
+        try:
+            resp, content = rc.post(
+                'create_network_function_config', body=body)
+        except:
+            LOG.error("create_%s -> request failed." % (name))
+
+    '''
+    def _put(self, context, tenant_id, name, **kwargs):
+        db = self._context(context, tenant_id)
+        context.__setattr__('service_info', db)
+        kwargs.update({'context': context})
+        resource = name
+        body = self._prepare_request_data(resource, kwargs)
+        try:
+            resp, content = rc.put('update_network_function_config', body=body)
+        except:
+            LOG.error("update_%s -> request failed." % (name))
+    '''
+
+    def _delete(self, context, tenant_id, name, **kwargs):
+        db = self._context(context, tenant_id)
+        context.__setattr__('service_info', db)
+        kwargs.update({'context': context})
+        resource = name
+        body = self._prepare_request_data(resource, kwargs)
+        try:
+            resp, content = rc.post('delete_network_function_config',
+                                    body=body, delete=True)
+        except:
+            LOG.error("delete_%s -> request failed." % (name))
+
+    def create_vip(self, context, vip):
+        self._post(context, vip['tenant_id'], 'vip', vip=vip)
+
+    '''
+    def update_vip(self, context, old_vip, vip):
```

Comment:
Is put going to be supported later? Remove for now and add it at that time?
Or uncomment the method with an empty body.